internal/templates/assisted-installer/template.go

```diff
@@ -126,6 +126,59 @@ spec:
   additionalNTPSources:
 {{ .Spec.AdditionalNTPSources | toYaml | indent 4 }}`
 
+const InfraEnvX86_64 = `apiVersion: agent-install.openshift.io/v1beta1
```

Comment:
I think it would be better to extend the current `InfraEnv` template instead of maintaining multiple, highly duplicated, versions.

Reply:
This would be my only comment as well. Extending / templating the existing `InfraEnv` should have the same end result.